.github/workflows/notify_ms_teams.yml

```diff
@@ -0,0 +1,72 @@
+name: Notify Teams on Push
```

Comment:
Can we follow the stands from `test.yml` or `sca.yml` please:
* File name one word `notify.yml`.
* Consistent step names from `test.yml` please.
* Ensure timezone, environment is setup in the similar manner please.

Reply:
I will update some to be repository etc.

For the custom `extract commit information` I could just rename to extract but it definitely takes away some of the context, are we okay with having multiple words in a step name?

Reply:
Yes please one word, can add some documentation about steps for context.
Reason I ask is when the pipeline is observed on VSCode multi words are concatenated.